hasjob/static/robots.txt

```diff
@@ -2,4 +2,6 @@ User-agent: *
 Disallow: /edit/*
 Disallow: /confirm/*
 Disallow: /withdraw/*
+Disallow: /close/*
+Disallow: /reopen/*
```

Comment:
These two shouldn't be required as we've moved all URLs to `/<domain>/<jobid>/<action>` format. The '/<action>/<jobid>` format is legacy and only for links from old emails (except in the case of `/new` and `/edit` for drafts).